utils/rpms/Makefile

```diff
@@ -6,26 +6,6 @@ PATCHES  = scons_local-$(VERSION).tar.$(SRC_EXT)
 #https://github.com/rpm-software-management/mock/issues/384
 MOCK_OPTIONS := --disablerepo=sclo*
 
-# override allows the caller to set a value and the we append to it
-#override PR_REPOS         +=
```

Comment:
Please don't remove these two lines.  They are there to remind people where they can add other component PR repos to build/test with.

Reply:
fixed